deploy/src/main/deploy/helm/values.yaml

```diff
@@ -1121,3 +1121,25 @@ prometheus-operator:
             ## Server name used to verify host name
             ##
             # serverName: ""
+
+
+# whether the Jaeger tracing component should be deployed
+deployJaeger: false
```

Comment:
Yes, I'll move the block above Prometheus operator.
Regarding standardization of names, instead of `deployJaeger`, Ì would suggest
````
jaeger:
  enabled: true
````
and keep all the jaeger specific configuration also below `jaeger`. (Alternative would be `tracing.jaeger.enabled`, but I would rather keep it simpler.)

For Prometheus, I guess we could use `prometheus.enabled` as a condition for the prometheus-operator dependency accordingly.